### PR TITLE
remove submodule and rename filename

### PR DESCRIPTION
**Pull request by acdo2002**
#113 
Following #114

Clean way to prevent any submodule contamination so I create a new branch for merge or any update only with those three tests.

### CODE REVIEW

File: src/test/PER_CUBE_HISTOGRAM.test.ts [hunk 1]
```diff
@@ -73,124 +65,107 @@ let assertItem: AssertItem = {
...
+            test(`return RASTER_TILE_DATA(Stream) and check total length `, async () => {
+                await Connection.send(CARTA.AddRequiredTiles, assertItem.addTilesReq);
+                await Connection.send(CARTA.SetCursor, assertItem.setCursor);
+                // await Connection.send(CARTA.SetSpatialRequirements, assertItem.setSpatialReq);
 
-                test(`REGION_HISTOGRAM_DATA.histograms.firt_bin_center = ${assertItem.regionHistogramData.histograms[0].firstBinCenter}`, () => {
-                    expect(RegionHistogramDataTemp.histograms[0].firstBinCenter).toBeCloseTo(assertItem.regionHistogramData.histograms[0].firstBinCenter, assertItem.precisionDigits);
-                });
+                ack = await Connection.stream(assertItem.addTilesReq.tiles.length + 3) as AckStream;
+                console.log(ack); // RasterTileData * 1 + SpatialProfileData * 1 + RasterTileSync *2 (start & end)
```
Comment (zarda) [inline src/test/PER_CUBE_HISTOGRAM.test.ts:107]: May not print this

File: src/test/PER_CUBE_HISTOGRAM_CANCELLATION.test.ts [hunk 2]
```diff
@@ -83,64 +89,88 @@ describe("PER_CUBE_HISTOGRAM_CANCELLATION tests: Testing the cancellation capabi
...
+                await Connection.send(CARTA.SetCursor, assertItem.setCursor);
+                // await Connection.send(CARTA.SetSpatialRequirements, assertItem.setSpatialReq);
 
-        test(`The second REGION_HISTOGRAM_DATA should arrive and REGION_HISTOGRAM_DATA.progress > previous one `, async () => {
-            RegionHistogramDataTemp = await Connection.receive(CARTA.RegionHistogramData);
-            regionHistogramProgress = RegionHistogramDataTemp.progress;
-            console.log(`Region Histogram Progress = ${regionHistogramProgress}`);
-        }, readFileTimeout);
+                ack = await Connection.stream(assertItem.addTilesReq.tiles.length + 3) as AckStream;
+                console.log(ack); // RasterTileData * 1 + SpatialProfileData * 1 + RasterTileSync *2 (start & end)
```
Comment (zarda) [inline src/test/PER_CUBE_HISTOGRAM_CANCELLATION.test.ts:116]: May not print this

File: src/test/PER_CUBE_HISTOGRAM_HDF5.test.ts [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+            }, openFileTimeout);
+
+            let ack: AckStream;
+            test(`return RASTER_TILE_DATA(Stream) and check total length `, async () => {
+                await Connection.send(CARTA.AddRequiredTiles, assertItem.addTilesReq);
+                await Connection.send(CARTA.SetCursor, assertItem.setCursor);
+                // await Connection.send(CARTA.SetSpatialRequirements, assertItem.setSpatialReq);
+
+                ack = await Connection.stream(assertItem.addTilesReq.tiles.length + 3) as AckStream;
+                console.log(ack); // RasterTileData * 1 + SpatialProfileData * 1 + RasterTileSync *2 (start & end)
```
Comment (zarda) [inline src/test/PER_CUBE_HISTOGRAM_HDF5.test.ts:107]: May not print this

File: src/test/PER_CUBE_HISTOGRAM.test.ts (hunk 1 above)
Comment (acdo2002) [reply src/test/PER_CUBE_HISTOGRAM.test.ts]: block the console.log

File: src/test/PER_CUBE_HISTOGRAM_CANCELLATION.test.ts (hunk 2 above)
Comment (acdo2002) [reply src/test/PER_CUBE_HISTOGRAM_CANCELLATION.test.ts]: block the console.log

File: src/test/PER_CUBE_HISTOGRAM_HDF5.test.ts (hunk 3 above)
Comment (acdo2002) [reply src/test/PER_CUBE_HISTOGRAM_HDF5.test.ts]: block the console.log

**Pull request merged**